Ginger/GingerCoreNET/Run/GingerExecutionEngine.cs

```diff
@@ -560,6 +559,8 @@ public void RunRunner(bool doContinueRun = false)
 
                     NotifyRunnerRunEnd(CurrentBusinessFlow.ExecutionFullLogFolder);
 
+                    ReportSkippedToSealights();
```

Comment:
1- please place it before "NotifyRunnerRunEnd"
2- call it "NotifyOnSkippedRunnerEntities" - and don't mention Sealights because this notify is generic, today only Sealights catch it but maybe in future other listeners will want to catch the skipped notify

Reply:
updated